ang/crmCaseType/list.html

```diff
@@ -13,9 +13,9 @@ <h1 crm-page-title>{{ts('Case Types')}}</h1>
   <table class="display">
     <thead>
     <tr>
-      <th>{{ts('Title')}}</th>
-      <th>{{ts('Name')}}</th>
-      <th>{{ts('Description')}}</th>
+      <th>{{:: ts('Title') }}</th>
+      <th>{{:: ts('Name') }}</th>
+      <th>{{:: ts('Description') }}</th>
       <th>{{ts('Enabled?')}}</th>
```

Comment:
I'm guessing a regex thing for line 19?

Reply:
Expanded the regex to include any character inside ts() as long as there are no variable substitutions.